src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/controller/ReconciliationController.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.controller;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.model.in.ReconciliationRequest;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.model.out.SuccessfulReconciliationResponse;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.service.ReconciliationService;
+
+import java.time.LocalDate;
+import java.util.UUID;
+import javax.validation.Valid;
+
+@ConditionalOnProperty("reconciliation.enabled")
+@RestController
+public class ReconciliationController {
+
+    private final ReconciliationService service;
+
+    public ReconciliationController(ReconciliationService service) {
+        this.service = service;
+    }
+
+    @PostMapping(
+        path = "/reform-scan/reconciliation-report/{date}",
```

Comment:
`/reform-scan` prefix is intended? asking just for clarification if it is

Reply:
the swagger spec has the path as `/reform-scan/reconciliation-report/{date}`